{"uuid":"b44bd9dd-cad4-4f22-a201-e5fc0b9bfdda","historyId":"8118a347ca9662f3e83ae183b42ba93a","fullName":"TestCases.Jobs_POST_Request.ValidateStatusLine","labels":[{"name":"package","value":"TestCases.Jobs_POST_Request"},{"name":"testClass","value":"TestCases.Jobs_POST_Request"},{"name":"testMethod","value":"ValidateStatusLine"},{"name":"parentSuite","value":"RestAssuredProject_Jobs_Api"},{"name":"suite","value":"Jobs API Test"},{"name":"subSuite","value":"TestCases.Jobs_POST_Request"},{"name":"host","value":"DESKTOP-4I6H6P7"},{"name":"thread","value":"4332@DESKTOP-4I6H6P7.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"epic","value":"03"}],"links":[],"name":"ValidateStatusLine","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [HTTP/1.1 200 OK] but found [HTTP/1.1 409 CONFLICT]","trace":"java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 409 CONFLICT]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat org.testng.Assert.assertEquals(Assert.java:665)\r\n\tat TestCases.Jobs_POST_Request.ValidateStatusLine(Jobs_POST_Request.java:134)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\r\n\tat org.testng.TestRunner.run(TestRunner.java:603)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\r\n\tat org.testng.TestNG.run(TestNG.java:1060)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","description":"Validate Status Line","steps":[],"attachments":[],"parameters":[],"start":1652978551250,"stop":1652978551251}